src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -54,6 +54,7 @@
         ("convnext", "ConvNextFeatureExtractor"),
         ("van", "ConvNextFeatureExtractor"),
         ("resnet", "ConvNextFeatureExtractor"),
+        ("regnet", "ConvNextFeatureExtractor"),
```

Comment:
We shouldn't add all these models with ConvNextFeatureExtractor, that seems weird to me..

Reply:
Which one would you rather use?